app/components/article-content.js

```diff
@@ -530,15 +530,14 @@ export default Component.extend(
     handleCollapsibleSectionHeaderClick(event) {
       const header = event.currentTarget;
 
+      if (event.target.closest('.section-edit-link')) {
```

Comment:
why do we need this here? maybe some comment would be helpful?

Reply:
to not collapse / uncollapse section if you click on edit link